agent/src/agent/pipeline/config/jython_scripts/send_watermark_and_monitoring_metrics.py

```diff
@@ -0,0 +1,53 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import time
+    import os
+    import sys
+    import calendar
+
+    from datetime import datetime
+
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+    import requests
+    import pytz
+finally:
+    sdc.importUnlock()
+
+
+def _get_watermark_delta(watermark):
+    tz = sdc.userParams['TIMEZONE'] if sdc.userParams['WATERMARK_IN_LOCAL_TIMEZONE'] == 'True' and sdc.userParams['TIMEZONE'] else 'UTC'
+    return str(calendar.timegm(datetime.now(pytz.timezone(tz)).timetuple()) - watermark)
+
+
+for record in sdc.records:
+    if not record.value['watermark'] or not record.value['schemaId']:
+        sdc.log.info('The pipeline doesn\'t have watermark or schemaId, skipping sending watermark')
+        continue
+
+    try:
+        # send Watermark
```

Comment:
Let's add a flag to pipeline `watermark_logs_enabled` (false by default) and if true add here before sending the watermark something like `sdc.log.info('Sending watermark: '   str(record.value))`. It will be useful for debugging pipelines and especially if there are different timezone settings